package.json

```diff
@@ -52,6 +52,7 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-es2015-rollup": "^1.1.1",
     "babel-register": "^6.3.13",
+    "backbone": ">=1.3.3 <1.4.0",
```

Comment:
Maybe it's better just use 1.3.3 as we did it for Mn. Bb are not using semver.  So 1.3.5 might have major changes or it's not critical for Radio?